modules/minimint-mint/src/config.rs

```diff
@@ -73,6 +77,25 @@ impl GenerateConfig for MintConfig {
 
         (mint_cfg, client_cfg)
     }
+
+    fn to_client_config(&self) -> Self::ClientConfig {
+        let pub_key: HashMap<Amount, AggregatePublicKey> = TieredMultiZip::new(
+            self.peer_tbs_pks
+                .iter()
+                .map(|(_, keys)| keys.iter())
+                .collect(),
+        )
+        .map(|(amt, keys)| {
+            // TODO: avoid this through better aggregation API allowing references or
```

Comment:
Was there something to follow after the "or"?

Reply:
Just a nit, merging anyway

Reply:
Ask the guy who wrote it ;) https://github.com/fedimint/fedimint/blob/f91fb2fa30e4e8e0e1a9ac8bc6f410ed07171bcd/modules/minimint-mint/src/lib.rs#L437